tests/ui/parallel-rustc/deadlock-issue-119785.rs

```diff
@@ -0,0 +1,74 @@
+//@ parallel-front-end
+//@ compile-flags: -Z threads=200
```

Comment:
Discussion: should we have a check that asserts `-Z threads` must *only* be used with `//@ parallel-front-end` unless it's checking the properties of `-Z threads` cli option itself?

Reply:
For clarification, I renamed this option to `parallel-front-end-robustness`, meaning that we only care about ice issues and not error output, so other tests using the `-Z threads` option will not rely on it.